refactor: remove plugins from subgraph (#571)
aragon/osx: packages/subgraph/CHANGELOG.md

## Split

> - Renamed Proposal to `ActionBatch` and made it immutable

Reviewer: ```suggestion
- Renamed `Proposal` to `ActionBatch` and made it immutable
```

- Added `deterministicId` as field to `ActionBatch` and `Action`. The `deterministicId` is a reference shared by the ActionId in plugins packages that can be known at the time of proposal execution.

> - The `deterministicId` and `Id` fields now include the `daoAddress` inside executeActions

Reviewer: ```suggestion
- The `deterministicId` and `Id` fields now include the `daoAddress` inside `executeActions`
```

> - Renamed `proposal` to `actionBatch` in `Action`, `ERC20Transfer`, `ERC721Transfer`, `ERC1155Transfer`, `NativeTransfer` and `DAO`

Reviewer: ```suggestion
- Renamed `Proposal` to `actionBatch` in `Action`, `ERC20Transfer`, `ERC721Transfer`, `ERC1155Transfer`, `NativeTransfer` and `DAO`
```

Author: Deliberately ignored, the original is correct

- Removed `startDate` and `endDate` from `ActionBatch` as they relate to proposals
- Removed `TokenVoting` `Multisig` `Admin` and `AddressList` from the schema
- Deleted the corresponding handlers any references to the above plugins

> - Removed the concept of wrapped ERC20 as these are only relevant in the context of tokenVoting

Reviewer: ```suggestion
- Removed the concept of wrapped ERC20 as these are only relevant in the context of the token voting plugin
```

## Split

- Renamed Proposal to `TransactionActions` and made it immutable
- Added deterministicID as field to `TransactionActions` and `Action`
- `Action` swaps `proposal` foreign key in place of `transactionAction`
- The `deterministicId` and `Id` fields now include the `daoAddress` inside executeActions
- Renamed `proposal` to `transactionActions` in ERC20Transfer, ERC721Transfer, ERC1155Transfer and NativeTransfer and DAO
- Removed `startDate` and `endDate` from `TransactionActions`

- Removed `TokenVoting` `Multisig` `Admin` and `AddressList` from the schema
- Deleted the corresponding handlers and references in code
- Removed the concept of wrapped ERC20 as these are only relevant in the context of tokenVoting

> - identifyAndFetchOrCreateERC20Entity is just `fetchOrCreateERC20Entity`

Reviewer: Can we write this better? I.e., format variables as code and specify where this was changed for traceability?
